### PR TITLE
bitvec_283

### PR DESCRIPTION
**Pull request by eurquhart1**
This resolves https://github.com/opencompl/ssa/issues/306.

### CODE REVIEW

File: SSA/Projects/InstCombine/AliveStatements.lean
Comment (eurquhart1) [inline SSA/Projects/InstCombine/AliveStatements.lean:1]: I'm not entirely sure how I can fix this, any advice would be appreciated!

Comment (tobiasgrosser) [reply SSA/Projects/InstCombine/AliveStatements.lean:1]: How nice. I fixed the newline issue in https://github.com/opencompl/ssa/pull/329. This change should be merged in 15 minutes. You can then just execute `git remote update; git merge origin/main` and the issue should be resolved in your branch.

**Review by tobiasgrosser — APPROVED** [review]
This looks great. Thank you @eurquhart1 .

File: SSA/Projects/InstCombine/AliveStatements.lean
Comment (tobiasgrosser) [reply SSA/Projects/InstCombine/AliveStatements.lean:1]: I did this for you.

Comment (eurquhart1) [reply SSA/Projects/InstCombine/AliveStatements.lean:1]: great, thank you very much. I'll remember this for next time

**Pull request merged**